tests/template_tests/test_nodelist.py

```diff
@@ -32,6 +33,21 @@ def test_ifchanged(self):
         self.assertEqual(len(vars), 1)
 
 
+class TextNodeReprTest(TestCase):
+
+    def test_textnode_repr(self):
+        engine = Engine()
```

Comment:
if kept as a separate class, I'd copy the `setUpClass` method from `NodelistTest`